api/server/router/network/network.go

```diff
@@ -6,21 +6,20 @@ import (
 	"github.com/docker/docker/api/server/httputils"
 	"github.com/docker/docker/api/server/router"
 	"github.com/docker/docker/api/server/router/local"
-	"github.com/docker/docker/daemon"
 	"github.com/docker/docker/errors"
 	"golang.org/x/net/context"
 )
 
 // networkRouter is a router to talk with the network controller
 type networkRouter struct {
-	daemon *daemon.Daemon
+	impl   Backend
```

Comment:
I feel `impl` is too "generic" name, `backend` should probably be better, or .. I don't know :sweat_smile: